app/src/main/java/com/nononsenseapps/feeder/ui/compose/searchfeed/SearchFeedScreen.kt

```diff
@@ -227,11 +228,13 @@ fun SearchFeedView(
     // If screen is opened from intent with pre-filled URL, trigger search directly
     LaunchedEffect(Unit) {
         if (results.item.isEmpty() && errors.item.isEmpty() && feedUrl.isNotBlank() &&
-            isValidUrl(
-                feedUrl,
-            )
+            (isValidUrl(feedUrl) || feedUrl.isNostrUri())
         ) {
-            onSearchCallback(sloppyLinkToStrictURLNoThrows(feedUrl))
+            onSearchCallback(
+                sloppyLinkToStrictURLNoThrows(
+                    if (feedUrl.isNostrUri()) "https://njump.me/$feedUrl" else feedUrl,
```

Comment:
`if (feedUrl.isNostrUri()) "https://njump.me/$feedUrl" else feedUrl` is repeated enough times that it should be moved to a simple getter